README.md

```diff
@@ -27,9 +27,9 @@ checkstyle.configurationFile=sun_checks.xml
 checkstyle.propertiesFile=
 pmd.enabled=true
 pmd.ruleSets=rulesets/java/android.xml,rulesets/java/basic.xml
-findbugs.enabled=true
```

Comment:
Add comment or information that findbug properties still work

Reply:
Done